aspnetcore/release-notes/aspnetcore-8.0.md

```diff
@@ -173,6 +173,39 @@ The main entry points to subsystems that don't work reliably with native AOT are
 
 :::image type="content" source="../fundamentals/aot/_static/top-level-annnotations.png" alt-text="Visual Studio window showing IL2026 warning message on the AddControllers method that says MVC doesn't currently support native AOT.":::
 
+### Minimal APIs and native AOT
+
+In order to make Minimal APIs compatible with native AOT, we're introducing the Request Delegate Generator (RDG). The RDG is a source generator that does what the <xref:Microsoft.AspNetCore.Http.RequestDelegateFactory> (RDF) does. That is, it turns the various `MapGet()`, `MapPost()`, and calls like them into <xref:Microsoft.AspNetCore.Http.RequestDelegate> instances associated with the specified routes. But rather than doing it in-memory in an application when it starts, the RDG does it at compile time and generates C# code directly into the project. This removes the runtime generation of this code and ensures that the types used in your APIs are statically analyzable by the native AOT tool-chain. This also ensures that required code is not trimmed away. We’re working to ensure that as many as possible of the Minimal API features are supported by the RDG and thus compatible with native AOT.
+
+The RDG is enabled automatically in your project when you enable publishing with native AOT. You can also manually enable RDG even when not using native AOT by setting <EnableRequestDelegateGenerator>true</EnableRequestDelegateGenerator> in your project file. This can be useful when initially evaluating your project’s readiness for native AOT, or to reduce the startup time of your application.
```

Comment:
```suggestion
The RDG is enabled automatically in a project when publishing with native AOT is enabled. RDG can be manually enabled even when not using native AOT by setting `<EnableRequestDelegateGenerator>true</EnableRequestDelegateGenerator>` in the project file. This can be useful when initially evaluating a project’s readiness for native AOT, or to reduce the startup time of an app.
```